test/data/stages/zip/zip.mpp.yaml

```
@@ -0,0 +1,57 @@
+version: '2'
+sources:
+  org.osbuild.inline: {}
+pipelines:
+  - mpp-import-pipelines:
+      path: ../../manifests/fedora-vars.ipp.yaml
+  - mpp-import-pipeline:
+      path: ../../manifests/fedora-build-v2.ipp.yaml
```

```suggestion
      path: ../manifests/fedora-vars.ipp.yaml
  - mpp-import-pipeline:
      path: ../manifests/fedora-build-v2.ipp.yaml
```

the `tar.mpp.yaml` file only has one `../` here. Do you know why this needs to be different?

---

Oh yeah, I meant to remove this after discussing other changes required in this PR. I think there might have been a directory structure change in OSBuild since running `osbuild mpp` results in an error that cannot find `fedora-vars.ipp.yaml` (and is why I added `../`). But yes, I plan on removing it.